extra/coreliquid_ctl

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python3
```

Comment:
What specific features does this script offer over `extra/yoda`?

Ideally, we should just improve `yoda` to support these additional features, unless there's something here that's totally unreasonable for any other device.

Reply:
The control methodology for this device is quite fundamentally different, so I wasn't sure how I could include it in yoda. The device has it's own PID controller, so the control script doesn't need to calculate or send fan speeds at all, it is sufficient to simply send the current temperature to the device and it will automatically control itself by the set profile. I just now realized that there is still an error in the way the current temp is sent and that a limitation from the manufacturer is that this method can only use one single temperature reading (usually cpu) to control all channels.

Another difference is that the device has named built in profiles, some of which have different limitations that I am not sure can be bypassed. For instance, custom fan speeds refuse to lower fan duties below 20%, while the SMART mode enables fan-stop for two of the three fans. I think the named profiles could quite easily be added to yoda, but there is no reason to if the control functions cannot be combined.

Reply:
I have now incorporated the necessary changes into yoda, which I think makes it slightly less clean but could be relatively simple to adapt for new devices that work similarly in the future.